binary_and_decimal.rb

```diff
@@ -12,6 +12,7 @@ def binary_to_decimal
       decimal_total += 2**i
     end
   end
+  #  puts "The binary number #{binary_num} = #{decimal_total}."
   return decimal_total
```

Comment:
The problem statement does ask you to print the binary and decimal values so what you had before with the print statements was correct.